# Warp
# Origin date: Tue, 12 Nov 2019 15:31:14 -0800
# Local date: Tue, 12 Nov 2019 15:31:14 -0800
# Commit hash: 06ab50e
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warpCparallel.cpython-37m-x86_64-linux-gnu.so
# Thu Jan  2 19:21:29 2020
# 24 processors
# import warp time 2.05572247505188 seconds
# For more help, type warphelp()
--------------------------------------------------------------------------
An MPI process has executed an operation involving a call to the
"fork()" system call to create a child process.  Open MPI is currently
operating in a condition that could result in memory corruption or
other system errors; your MPI job may hang, crash, or produce silent
data corruption.  The use of fork() (or system() or other calls that
create child processes) is strongly discouraged.  

The process that invoked fork was:

  Local host:          eu-a6-012-07 (PID 67523)
  MPI_COMM_WORLD rank: 3

If you are *absolutely sure* that your application will successfully
and correctly survive a call to fork(), you may disable this warning
by setting the mpi_warn_on_fork MCA parameter to 0.
--------------------------------------------------------------------------
('l_fftw_fort', False)
WARNING: module txphysics is not accessible.
WARNING: module desorb is not accessible.
0.0007224415584921976
Simulating ECLOUD in a crab cavity
[eu-a6-012-07:67519] 23 more processes have sent help message help-mpi-runtime.txt / mpi_init:warn-fork
[eu-a6-012-07:67519] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
Using cosine_3D emission angle distribution.
Secondary emission model: ECLOUD E0=150.0000 s=1.3500
#############################################################
Temp distribution found. Reloading it as initial distribution
#############################################################
Plot file name complete_sim.006.cgm
 ***  particle simulation package W3D generating
 ---  Resetting lattice array sizes
 ---  Allocating space for particles
 ---  Loading particles
 ---  Setting charge density
 ---  done
 ---  Allocating Win_Moments
 ---  Allocating Z_Moments
 ---  Allocating Lab_Moments
 Atomic number of ion =  1.0073E+00
 Charge state of ion  =  1.0000E+00
 Initial X, Y emittances =  0.0000E+00,   0.0000E+00 m-rad
 Initial X,Y envelope radii  =  0.0000E+00,   0.0000E+00 m
 Initial X,Y envelope angles =  0.0000E+00,   0.0000E+00 rad
 Input beam current =  0.0000E+00 amps
 Current density =  0.0000E+00 amps/m**2
 Charge density =  0.0000E+00 Coul/m**3
 Number density =  0.0000E+00
 Plasma frequency     =  0.0000E+00 1/s
    times dt          =  0.0000E+00
    times quad period =  0.0000E+00
 Plasma period        =  6.2832E+36 s
 X-, Y-Thermal Velocities     =  0.0000E+00,   0.0000E+00 m/s
    times dt                  =  0.0000E+00,   0.0000E+00 m
    times dt/dx, dt/dy (X, Y) =  0.0000E+00,   0.0000E+00
 X-, Y-Debye Wavelengths  =  0.0000E+00,   0.0000E+00 m
    over dx, dy (X and Y) =  0.0000E+00,   0.0000E+00
 Longitudinal thermal velocity (rms) =  0.0000E+00 m/s
    times dt                   =  0.0000E+00 m
    times dt/dz                =  0.0000E+00
 Longitudinal Debye wavelength =  0.0000E+00 m
    over dz                    =  0.0000E+00
 Beam velocity =  0.0000E+00 m/s
    over c     =  0.0000E+00
 Kinetic energy =  0.0000E+00 eV
 Weight of simulation particles =  1.0000E+00
 Number of simulation particles =            0
 Number of real particles =  0.0000E+00
 Total mass =  0.0000E+00 kg
 Total charge =  0.0000E+00 Coul
 Generalized perveance =  0.0000E+00
 Characteristic current =  3.1297E+07 amps
 Budker parameter =  0.0000E+00
 Timestep size dt =  2.5000E-11 s
 Tune length =  0.0000E+00
 Undep. X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Undep. X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 Undep. X-, Y-Betatron wavelengths  =  0.0000E+00,   0.0000E+00 m
 Dep.   X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Dep.   X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 X-, Y-Tune Depressions (sigma/sigma0) =  0.0000E+00,   0.0000E+00
 Space charge wave velocity =  0.0000E+00 m/s
 Effective wall radius =  2.1355E-01 m
 Geometric factor =  7.9805E+01
 X-, Y-Emittance over Space charge forces =  0.0000E+00,   0.0000E+00
 Number of grid points in x =      200
 Number of grid points in y =      200
 Number of grid points in z =      100
 Grid spacing in x =  1.5100E-03 m
 Grid spacing in y =  1.2200E-03 m
 Grid spacing in z =  6.0200E-03 m
 Bend radius =  0.0000E+00 m
 Bending field =  0.0000E+00 T
 Bend length =  0.0000E+00 m
 Straight section length =  0.0000E+00 m
 Z at start of first bend =  0.0000E+00 m
it =      0 time = 0.0000E+00 pz = 0.0000E+00 ese = 0.0000E+00 ek = 0.0000E+00 et = 0.0000E+00
 ***  particle simulation package W3D running
Multigrid: Error converged to   2.760E-07 in     8 v-cycles             
it =      1 time = 2.5000E-11 pz = 0.0000E+00 ese = 7.1112E-12 ek = 0.0000E+00 et = 7.1112E-12
Multigrid: Error converged to   5.274E-07 in    14 v-cycles             
it =      2 time = 5.0000E-11 pz = 0.0000E+00 ese = 3.0931E-11 ek = 0.0000E+00 et = 3.0931E-11
Saving a checkpoint!
===========================
Bunch passage: 7
Number of electrons in the dipole: 36050679
Number of macroparticles: 7820969
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
mpirun: killing job...

Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 76, in update_outputs
    + ew.get_density(l_dividebyvolume=0)[:,:,:])
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 77, in update_outputs
    self.numelecs[n_step] = np.sum(elecs_density)
  File "<__array_function__ internals>", line 6, in sum
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 78, in update_outputs
    self.numelecs_tot[n_step] = np.sum(elecs_density_tot)
  File "<__array_function__ internals>", line 6, in sum
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/numpy/core/fromnumeric.py", line 2182, in sum
    initial=initial, where=where)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/numpy/core/fromnumeric.py", line 90, in _wrapreduction
    return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
KeyboardInterrupt
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 306, in all_steps
    self.step(self.tot_nsteps)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 292, in step
    self.n_step) 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 78, in update_outputs
    self.numelecs_tot[n_step] = np.sum(elecs_density_tot)
  File "<__array_function__ internals>", line 6, in sum
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/numpy/core/fromnumeric.py", line 2182, in sum
    initial=initial, where=where)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/numpy/core/fromnumeric.py", line 90, in _wrapreduction
    return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
KeyboardInterrupt
# Warp
# Origin date: Tue, 12 Nov 2019 15:31:14 -0800
# Local date: Tue, 12 Nov 2019 15:31:14 -0800
# Commit hash: 06ab50e
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warpCparallel.cpython-37m-x86_64-linux-gnu.so
# Fri Jan  3 06:19:45 2020
# 24 processors
# import warp time 2.6127002239227295 seconds
# For more help, type warphelp()
--------------------------------------------------------------------------
An MPI process has executed an operation involving a call to the
"fork()" system call to create a child process.  Open MPI is currently
operating in a condition that could result in memory corruption or
other system errors; your MPI job may hang, crash, or produce silent
data corruption.  The use of fork() (or system() or other calls that
create child processes) is strongly discouraged.  

The process that invoked fork was:

  Local host:          eu-a6-007-22 (PID 39262)
  MPI_COMM_WORLD rank: 13

If you are *absolutely sure* that your application will successfully
and correctly survive a call to fork(), you may disable this warning
by setting the mpi_warn_on_fork MCA parameter to 0.
--------------------------------------------------------------------------
('l_fftw_fort', False)
WARNING: module txphysics is not accessible.
WARNING: module desorb is not accessible.
0.0007224415584921976
Simulating ECLOUD in a crab cavity
[eu-a6-007-22:39243] 23 more processes have sent help message help-mpi-runtime.txt / mpi_init:warn-fork
[eu-a6-007-22:39243] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Using cosine_3D emission angle distribution.
Secondary emission model: ECLOUD E0=150.0000 s=1.3500
#############################################################
Temp distribution found. Reloading it as initial distribution
#############################################################
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py:62: UserWarning: there was an error importing gist; if this is a problem, restart python and type 'import gist' for details, otherwise Warp will run OK but with no graphics
  warnings.warn("there was an error importing gist; if this is a problem, restart python and type 'import gist' for details, otherwise Warp will run OK but with no graphics")
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 88, in <module>
    sim = warp_pyecloud_sim(**kwargs)
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 114, in __init__
    electron_background_dist = self.load_elec_density()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 360, in load_elec_density
    self.n_step = dict_init_dist['n_step'][0][0]
KeyError: 'n_step'
--------------------------------------------------------------------------
mpirun noticed that the job aborted, but has no info as to the process
that caused that situation.
--------------------------------------------------------------------------
# Warp
# Origin date: Thu, 2 Jan 2020 15:21:51 -0800
# Local date: Thu, 2 Jan 2020 15:21:51 -0800
# Commit hash: d24b7d7
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py
# /cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warpCparallel.cpython-37m-x86_64-linux-gnu.so
# Sun Jan  5 10:45:48 2020
# 24 processors
# import warp time 2.7406699657440186 seconds
# For more help, type warphelp()
--------------------------------------------------------------------------
An MPI process has executed an operation involving a call to the
"fork()" system call to create a child process.  Open MPI is currently
operating in a condition that could result in memory corruption or
other system errors; your MPI job may hang, crash, or produce silent
data corruption.  The use of fork() (or system() or other calls that
create child processes) is strongly discouraged.  

The process that invoked fork was:

  Local host:          eu-a6-009-03 (PID 51361)
  MPI_COMM_WORLD rank: 0

If you are *absolutely sure* that your application will successfully
and correctly survive a call to fork(), you may disable this warning
by setting the mpi_warn_on_fork MCA parameter to 0.
--------------------------------------------------------------------------
('l_fftw_fort', False)
WARNING: module txphysics is not accessible.
WARNING: module desorb is not accessible.
0.0007224415584921976
Simulating ECLOUD in a crab cavity
[eu-a6-009-03:51356] 23 more processes have sent help message help-mpi-runtime.txt / mpi_init:warn-fork
[eu-a6-009-03:51356] Set MCA parameter "orte_base_help_aggregate" to 0 to see all help / error messages
Using cosine_3D emission angle distribution.
Secondary emission model: ECLOUD E0=150.0000 s=1.3500
Plot file name complete_sim.011.cgm
 ***  particle simulation package W3D generating
 ---  Resetting lattice array sizes
 ---  Allocating space for particles
 ---  Loading particles
 ---  Setting charge density
 ---  done
 ---  Allocating Win_Moments
 ---  Allocating Z_Moments
 ---  Allocating Lab_Moments
 Atomic number of ion =  1.0073E+00
 Charge state of ion  =  1.0000E+00
 Initial X, Y emittances =  0.0000E+00,   0.0000E+00 m-rad
 Initial X,Y envelope radii  =  0.0000E+00,   0.0000E+00 m
 Initial X,Y envelope angles =  0.0000E+00,   0.0000E+00 rad
 Input beam current =  0.0000E+00 amps
 Current density =  0.0000E+00 amps/m**2
 Charge density =  0.0000E+00 Coul/m**3
 Number density =  0.0000E+00
 Plasma frequency     =  0.0000E+00 1/s
    times dt          =  0.0000E+00
    times quad period =  0.0000E+00
 Plasma period        =  6.2832E+36 s
 X-, Y-Thermal Velocities     =  0.0000E+00,   0.0000E+00 m/s
    times dt                  =  0.0000E+00,   0.0000E+00 m
    times dt/dx, dt/dy (X, Y) =  0.0000E+00,   0.0000E+00
 X-, Y-Debye Wavelengths  =  0.0000E+00,   0.0000E+00 m
    over dx, dy (X and Y) =  0.0000E+00,   0.0000E+00
 Longitudinal thermal velocity (rms) =  0.0000E+00 m/s
    times dt                   =  0.0000E+00 m
    times dt/dz                =  0.0000E+00
 Longitudinal Debye wavelength =  0.0000E+00 m
    over dz                    =  0.0000E+00
 Beam velocity =  0.0000E+00 m/s
    over c     =  0.0000E+00
 Kinetic energy =  0.0000E+00 eV
 Weight of simulation particles =  1.0000E+00
 Number of simulation particles =            0
 Number of real particles =  0.0000E+00
 Total mass =  0.0000E+00 kg
 Total charge =  0.0000E+00 Coul
 Generalized perveance =  0.0000E+00
 Characteristic current =  3.1297E+07 amps
 Budker parameter =  0.0000E+00
 Timestep size dt =  2.5000E-11 s
 Tune length =  0.0000E+00
 Undep. X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Undep. X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 Undep. X-, Y-Betatron wavelengths  =  0.0000E+00,   0.0000E+00 m
 Dep.   X-, Y-Betatron frequencies  =  6.2832E+36,   6.2832E+36 1/s
 Dep.   X-, Y-Betatron periods      =  0.0000E+00,   0.0000E+00 s
 X-, Y-Tune Depressions (sigma/sigma0) =  0.0000E+00,   0.0000E+00
 Space charge wave velocity =  0.0000E+00 m/s
 Effective wall radius =  2.1355E-01 m
 Geometric factor =  7.9805E+01
 X-, Y-Emittance over Space charge forces =  0.0000E+00,   0.0000E+00
 Number of grid points in x =      200
 Number of grid points in y =      200
 Number of grid points in z =      100
 Grid spacing in x =  1.5100E-03 m
 Grid spacing in y =  1.2200E-03 m
 Grid spacing in z =  6.0200E-03 m
 Bend radius =  0.0000E+00 m
 Bending field =  0.0000E+00 T
 Bend length =  0.0000E+00 m
 Straight section length =  0.0000E+00 m
 Z at start of first bend =  0.0000E+00 m
it =      0 time = 0.0000E+00 pz = 0.0000E+00 ese = 0.0000E+00 ek = 0.0000E+00 et = 0.0000E+00
 ***  particle simulation package W3D running
Multigrid: Error converged to   2.777E-07 in     8 v-cycles             
it =      1 time = 2.5000E-11 pz = 0.0000E+00 ese = 6.9535E-12 ek = 0.0000E+00 et = 6.9535E-12
Multigrid: Error converged to   4.849E-07 in    13 v-cycles             
it =      2 time = 5.0000E-11 pz = 0.0000E+00 ese = 3.0175E-11 ek = 0.0000E+00 et = 3.0175E-11
===========================
Bunch passage: 0
Number of electrons in the dipole: 19990761
Number of macroparticles: 4198060
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 1
Number of electrons in the dipole: 20750142
Number of macroparticles: 4381700
Previous passage took 1637s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 2
Number of electrons in the dipole: 22000933
Number of macroparticles: 4627963
Previous passage took 1693s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 3
Number of electrons in the dipole: 23488189
Number of macroparticles: 5031577
Previous passage took 1726s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 4
Number of electrons in the dipole: 25344599
Number of macroparticles: 5530955
Previous passage took 1782s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 5
Number of electrons in the dipole: 27936004
Number of macroparticles: 6165527
Previous passage took 1854s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 6
Number of electrons in the dipole: 31594742
Number of macroparticles: 7102044
Previous passage took 1938s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 6202450
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 13
Start SOFT regeneration. N_mp=6202450 Nel_tot=2.66e+07, En_tot = 2.74e+08
Applied correction factor = 3.103023e+00
Done SOFT regeneration. N_mp=1998771 Nel_tot=2.66e+07 En_tot=2.74e+08
Saving a checkpoint!
===========================
Bunch passage: 7
Number of electrons in the dipole: 36712930
Number of macroparticles: 4131540
Previous passage took 2077s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 8
Number of electrons in the dipole: 43755287
Number of macroparticles: 4782596
Previous passage took 1593s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 9
Number of electrons in the dipole: 53351543
Number of macroparticles: 5554744
Previous passage took 1686s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 10
Number of electrons in the dipole: 66546283
Number of macroparticles: 6612307
Previous passage took 1870s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 6653918
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 39
Start SOFT regeneration. N_mp=6653918 Nel_tot=7.81e+07, En_tot = 7.80e+08
Applied correction factor = 3.327241e+00
Done SOFT regeneration. N_mp=2000136 Nel_tot=7.81e+07 En_tot=7.81e+08
Saving a checkpoint!
===========================
Bunch passage: 11
Number of electrons in the dipole: 84665441
Number of macroparticles: 3380433
Previous passage took 2066s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 12
Number of electrons in the dipole: 109395544
Number of macroparticles: 4180488
Previous passage took 1538s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 13
Number of electrons in the dipole: 142932295
Number of macroparticles: 5091919
Previous passage took 1680s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 14
Number of electrons in the dipole: 187779276
Number of macroparticles: 6298537
Previous passage took 1868s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 6907235
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 121
Start SOFT regeneration. N_mp=6907235 Nel_tot=2.42e+08, En_tot = 2.37e+09
Applied correction factor = 3.451810e+00
Done SOFT regeneration. N_mp=2000742 Nel_tot=2.42e+08 En_tot=2.37e+09
Saving a checkpoint!
===========================
Bunch passage: 15
Number of electrons in the dipole: 246795078
Number of macroparticles: 2987780
Previous passage took 2103s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 16
Number of electrons in the dipole: 322687256
Number of macroparticles: 3789203
Previous passage took 1514s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 17
Number of electrons in the dipole: 417384181
Number of macroparticles: 4661405
Previous passage took 1660s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 18
Number of electrons in the dipole: 531067134
Number of macroparticles: 5734661
Previous passage took 1830s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 6303382
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 329
Start SOFT regeneration. N_mp=6303382 Nel_tot=6.59e+08, En_tot = 6.65e+09
Applied correction factor = 3.150460e+00
Done SOFT regeneration. N_mp=2001267 Nel_tot=6.59e+08 En_tot=6.66e+09
Saving a checkpoint!
===========================
Bunch passage: 19
Number of electrons in the dipole: 662164138
Number of macroparticles: 2726638
Previous passage took 2053s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 20
Number of electrons in the dipole: 807243263
Number of macroparticles: 3437086
Previous passage took 1503s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 21
Number of electrons in the dipole: 962129228
Number of macroparticles: 4090603
Previous passage took 1596s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 22
Number of electrons in the dipole: 1121979976
Number of macroparticles: 4793712
Previous passage took 1688s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 23
Number of electrons in the dipole: 1282488443
Number of macroparticles: 5543453
Previous passage took 1831s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Saving a checkpoint!
===========================
Bunch passage: 24
Number of electrons in the dipole: 1439294268
Number of macroparticles: 6331033
Previous passage took 1938s
10% of bunch passage
20% of bunch passage
30% of bunch passage
40% of bunch passage
50% of bunch passage
60% of bunch passage
70% of bunch passage
80% of bunch passage
90% of bunch passage
Number of macroparticles: 6678573
MAXIMUM LIMIT OF MPS HAS BEEN RACHED
New nel_mp_ref = 794
Start SOFT regeneration. N_mp=6678573 Nel_tot=1.59e+09, En_tot = 1.78e+10
Applied correction factor = 3.339543e+00
Done SOFT regeneration. N_mp=2000493 Nel_tot=1.59e+09 En_tot=1.78e+10
Saving a checkpoint!
===========================
Bunch passage: 25
Number of electrons in the dipole: 1590513729
Number of macroparticles: 2467578
Previous passage took 2044s
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
/cluster/home/glorenzo/sim_workspace_mpi_py3/venvs/py3/lib/python3.7/site-packages/warp/warp.py:62: UserWarning: there was an error importing gist; if this is a problem, restart python and type 'import gist' for details, otherwise Warp will run OK but with no graphics
  warnings.warn("there was an error importing gist; if this is a problem, restart python and type 'import gist' for details, otherwise Warp will run OK but with no graphics")
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
Traceback (most recent call last):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/simulations/scan_efield_cavity/simulations/complete_cavity_Emax_5.0/complete_sim.py", line 90, in <module>
    sim.all_steps()
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 302, in all_steps
    def all_steps(self):
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/warp_pyecloud_sim.py", line 276, in step
    self.saver.dump_outputs(self.chamber.xmin, self.chamber.xmax, 
  File "/cluster/home/glorenzo/sim_workspace_mpi_py3/WarPyECLOUD/saver.py", line 89, in dump_outputs
    dict_out['numelecs'] = self.numelecs
IndexError: index 25 is out of bounds for axis 0 with size 25
--------------------------------------------------------------------------
mpirun noticed that the job aborted, but has no info as to the process
that caused that situation.
--------------------------------------------------------------------------
